velox/docs/monthly-updates/july-2022.rst

```diff
@@ -0,0 +1,58 @@
+****************
+July 2022 Update
+****************
+
+Documentation
+=============
+
+* Add documentation for Presto related :doc:`/functions/json`.
+
+Core Library
+============
+
+* Add `ApproxMostFrequentStreamSummary` that approximately computes the top frequent values from a large stream.
+* Add support for cast from a JSON type value to ARRAYS or MAPS of JSON type values.
+* Add Parquet writer implemented over Apache Arrow Parquet writer.
+* Add NegatedBytesRange filter.
+* Add PlanNode, PlanBuilder, registry support for Window operators.
+* Add support for DECIMAL type Variant values.
+
+Presto Functions
+================
+
+* Add :func:`array_sort`, :func:`combinations` functions.
+* Add :func:`approx_most_frequent` aggregate function.
+* Extend aggregate functions :func:`min_by`, :func:`max_by` to support VARCHAR.
+
+Hive Connector
+==============
+
+* Add NegatedBigintRange filters support to DWRF reader.
```

Comment:
There is a duplicate item in Core Library section. Let's remove it from there and maybe clarify this one here:

```
* Add NegatedBytesRange filter to optimize NOT IN ('a', 'b', 'c') filters on string columns pushed down into DWRF reader.
```

Reply:
I just used `NOT IN` since we mention `string columns`.